packages/system/keycloak-configure/templates/configure-kk.yaml

```
@@ -225,3 +225,16 @@ spec:
   realmRef:
     name: keycloakrealm-cozy
     kind: ClusterKeycloakRealm
+
+---
+
+apiVersion: v1.edp.epam.com/v1
+kind: KeycloakRealmGroup
+metadata:
+  name: cozystack-cluster-admin
+  namespace: cozy-dashboard
```

Let's move it to cozy-system namespace

```suggestion
  namespace: cozy-system
```